tox.ini

```diff
@@ -47,7 +47,9 @@ changedir = {toxinidir}
 extras = docs
 setenv =
     HOME = {envtmpdir}
-commands = sphinx-build docs docs{/}_build{/}html -W -n --keep-going -b html {posargs}
+commands =
+    sphinx-build docs docs{/}_build{/}html -W -n --keep-going -b html {posargs}
+    echo "Troubleshooting guide: https://docs.plasmapy.org/en/latest/contributing/doc_guide.html#documentation-tools"
```

Comment:
Wow.

I mean, wow. That's a bloody great idea, massive kudos.

Do you think we could add the same thing to test runs, for the testing guide?

Reply:
> Do you think we could add the same thing to test runs, for the testing guide?

Ideally, yes!  That'll be a bit trickier since there's a much broader range of things that can go wrong, but I'll create an issue on it.